doc/dynamicproxy.md

```diff
@@ -0,0 +1,20 @@
+# Dynamically generated proxies
+
+When a JSON-RPC server's API is expressed as a .NET interface, StreamJsonRpc can dynamically create a proxy that implements that interface
```

Comment:
In markdown, word wrapping isn't necessary since the renderer will do it. But people tend to still prefer manually wrapping lines around 120 characters. That's what's happened here. I ended the line after `interface` so that it didn't run too long, but it happens to be slightly longer than github's width, so it looks like a new line with just interface, but it actually is part of the prior line.